src/main/java/pl/allegro/tech/build/axion/release/domain/hooks/CommitHookAction.java

```diff
@@ -24,7 +24,7 @@ public String call(Object... args) {
 
     @Override
     public void act(HookContext hookContext) {
-        String message = (customAction.call(hookContext.getCurrentVersion(), hookContext.getPosition())).toString();
+        String message = (customAction.call(hookContext.getReleaseVersion(), hookContext.getPosition())).toString();
```

Comment:
typo?

Reply:
I saw it's deprecated and `getReleaseVersion()` uses the same underlying private field:

```java
    public String getReleaseVersion() {
        return currentVersion;
    }

    @Deprecated
    public String getCurrentVersion() {
        return currentVersion;
    }
```

Should I revert this or leave at it is?

Reply:
no, it's ok